pages/tokens/[id].tsx

```diff
@@ -8,18 +8,30 @@ import { CONFIG } from '@/const/meta'
 
 const DATA_CACHE_TIME_SECONDS = 10 * 60 // 10 minutes
 
-export type TokenDetailPageProps = TokenDetailProps
+export type TokenDetailPageProps = TokenDetailProps & {
+  currentUrl: string;
+}
 
-export default function TokenDetailsPage({ token }: TokenDetailPageProps) {
-  const { name, symbol, metaDescription } = token
+export default function TokenDetailsPage({ token, currentUrl }: TokenDetailPageProps) {
+  const { name, symbol, metaDescription, change24h, priceUsd } = token
+  const change24 = parseFloat(change24h)
+  const change24hFormatted = change24.toFixed(2)
+  const isIncrease = parseFloat(change24h) >= 0;
+  const changePlusMinus = isIncrease ? '+' : '-';
```

Comment:
You might not want to display the `-` as the number itself will have the signal when negative:

![image](https://github.com/cowprotocol/cow-fi/assets/43217/cb2ea758-4a49-4249-851a-1b9b90c11ce9)

Reply:
Removed, but now for me doesn't show the minus or plus.

Reply:
Fixed for page title, meta description still has double `-`

![image](https://github.com/cowprotocol/cow-fi/assets/43217/25108248-14d6-4820-887a-40eaee7da8c5)

Reply:
@alfetopito Removed, but again, for me now there is no + or -. Not sure why for you it's doable. As if code is rendered differently.

Reply:
Yeah, you removed both, and I'm fine with that.

Now, only negative values have the sign
![image](https://github.com/cowprotocol/cow-fi/assets/43217/82136d1d-6e68-4d68-96f6-3d3653f88f49)
![image](https://github.com/cowprotocol/cow-fi/assets/43217/06adc689-8a57-4c1a-b69e-4a9f2cb1da3c)